packages/hyperion-autologging/src/ALReactUtils.ts

```diff
@@ -46,9 +46,24 @@ type ReactInternalFiberType = Readonly<{
 }>;
 
 const getReactInternalFiber = (element: Element): ReactInternalFiber | null => {
-  const key = Object.keys(element).find(key => key.startsWith('__reactFiber$'));
+  let fiberKey: string | null = null;
+  let listeningKeyFound = false;
+  // Iterate the keys of the element, looking for react specific props
+  // If this is a reactListening node, then fiber will not be available, attempt to grab fiber from the parent element.
+  for (const key of Object.keys(element)) {
+    if (key.startsWith('__reactFiber$')) {
```

Comment:
Btw, I think we can make this much faster by finding the hash value that is used for the application. I think that value is fixed during lifetime of the application, e.g. `__reactFiber$sftyze5gh5`. So, may be if we don't know the full name, search for it once, but if we cannot find that, then do the search in the loop.

Reply:
This is a good idea!  And yes that seems to be the case,  makes this function a little more involved but I think any perf wins will be worth it.   It seems to work as expected,  please see latest push